cdap-data-fabric/src/main/java/io/cdap/cdap/data2/dataset2/lib/table/leveldb/LevelDBTableCore.java

```diff
@@ -173,10 +173,73 @@ public void persist(Map<byte[], ? extends Map<byte[], byte[]>> changes, long ver
     db.write(batch, service.getWriteOptions());
   }
 
+  /**
+   * Write the value at the target row and column with the max version {@link KeyValue.LATEST_TIMESTAMP}.
+   * as a result it hides any value written with equal or smaller version.
+   */
+  public void put(byte[] row, byte[] column, byte[] value) throws IOException {
```

Comment:
Why is it needed? What would happen if there are multiple put calls? This value is also used in querying, is it safe to use it for writing?

Reply:
Used by callers who don't care about versioning values. They can put(row, col, val). Each put call overwrites values by previous put calls.

Reply:
Can we use some other magic value? This LATEST_TIMESTAMP is used to define ranges and I am worrying that we may break some inclusive/exclusive queries that use LATEST_TIMESTAMP on it's bounds and don't care much about edge inclusion. e.g. can we use `0` ?

Reply:
LATEST_TIMESTAMP is the right one to use, existing code is built on the assumption latest_timestamp corresponding to the most recent version (i.e. the version of value to be returned upon read). This ensure that any write overwrites existing value.